benchmarks/setup/default/zeebe-values.yaml

```diff
@@ -84,10 +84,10 @@ env:
 resources:
   limits:
     cpu: 5
-    memory: 12Gi
+    memory: 4Gi
   requests:
-    cpu: 5
-    memory: 12Gi
+    cpu: 2
```

Comment:
:wrench: In the past we pegged requests to limits for brokers because you are never guaranteed the limit. It's possibly you only get the request and nothing else, so you should always plan as if you only got the request.

Reply:
That is true, so your suggestion would be to adjust the requests? Or the limits?

Reply:
Adjust the requests to what we think is the minimum required resources for the worst case scenario. For brokers, that means one broker is leader for all its partitions at the same time. For gateways...I don't know...it's serving all the requests? Is that even likely with a load balancer? :thinking: